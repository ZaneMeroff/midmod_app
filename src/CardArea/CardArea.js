import React from 'react';
import Card from '../Card/Card';
import './CardArea.css'

const CardArea = props => {


  return (
    <section>
      
    </section>
  )

}

export default CardArea;
