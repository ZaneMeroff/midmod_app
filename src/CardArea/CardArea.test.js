import React from 'react';
import ReactDOM from 'react-dom';
import CardArea from './CardArea';
import { shallow } from 'enzyme';

describe('Card', () => {
  let wrapper;
  let mockRes = [{
    name: 'fakeName',
    date: 'fakeDate',
    time: 'fakeTime',
    number: 'fakeNumber',
    id: 'fakeId',
  }]

  beforeEach(() => {
    wrapper = shallow(
      <CardArea
        cards={mockRes}
       />
     )
  })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<CardArea />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})
