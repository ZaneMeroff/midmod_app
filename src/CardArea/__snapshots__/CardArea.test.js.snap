// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<div
  id="card-area"
>
  <Card
    date="fakeDate"
    id="fakeId"
    name="fakeName"
    number="fakeNumber"
    time="fakeTime"
  />
</div>
`;
